dist/google-music.js

```diff
@@ -338,9 +352,9 @@ proto.hooks = {
             var durationStr = that.doc.getElementById(SELECTORS.playback.sliderId).getAttribute('aria-valuemax');
             var duration = parseInt(durationStr, 10);
 
-            title = (title) ? title.innerText : 'Unknown';
-            artist = (artist) ? artist.innerText : 'Unknown';
-            album = (album) ? album.innerText : 'Unknown';
+            title = (title) ? (title.innerText || title.textContent) : 'Unknown';
```

Comment:
This comment hasn't been taken care of =/

We can safely drop IE <=8 support since we rely on `dataset` and other nicities